src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -19,6 +19,8 @@ public class CliSyntax {
     public static final Prefix PREFIX_EXAM_DESCRIPTION = new Prefix("ex/");
     public static final Prefix PREFIX_EXAM_DATE = new Prefix("ed/");
     public static final Prefix PREFIX_DEADLINE = new Prefix("dl/");
+    public static final Prefix PREFIX_MOD_NAME = new Prefix("mn/");
+    public static final Prefix PREFIX_MOD_CREDIT = new Prefix("mc/");
```

Comment:
Same as above comment.

Reply:
I would think that mc would be the most appropriate and intuitive name for module credit though. Maybe can think of a shorter one-character name in the future